client/module/radio.lua

```diff
@@ -143,10 +143,15 @@ end)
 --- but you can integrate the below state bag to your death resources.
 --- LocalPlayer.state:set('isDead', true or false, false)
 function isDead()
+	local playerPed = PlayerPedId()
 	if LocalPlayer.state.isDead then
 		return true
 	elseif IsPlayerDead(PlayerId()) then
 		return true
+	elseif IsEntityPlayingAnim(playerPed, 'dead', 'dead_a', 3) then
```

Comment:
If you use a custom death animation you should instead just use the LocalPlayer state, its quite simple and prevents there having to be more and more and more else ifs here

Reply:
I use state bags myself but I added these for the people that don’t use state bags, I don’t see any harm in adding it for the less experienced people that are too stubborn to learn by reading the state bag documentation or the comment I added in my last PR for this death check. It’s up to you though, if you still want it removed that’s okay.

Reply:
I would rather not have it, eventually™️ the plan is to completely remove exports from this project all together and completely use state bags

Reply:
Removed 👍😄